README.md

```diff
@@ -349,16 +349,21 @@ for the actual, executable background task.
 
 # Testing
 
-It's possible to use Sqlite to emulate SQS when running the tests.
+It's possible to use SQLite to emulate SQS when running the tests.
 
 To do so, change the value of the env SQS_QUEUE_URL as the following examples:
 
 ```
-# to use a file
-SQS_QUEUE_URL=sqlite:/path/filename.sqlite3
+# To use a file
 
-# to use the memory
-SQS_QUEUE_URL=sqlite:/memory
+# It saves the data in /tmp/filename.sqlite3)
+SQS_QUEUE_URL=sqlite3://tmp/filename.sqlite3
+
+# It saves the data in /var/tmp/filename.sqlite3)
+SQS_QUEUE_URL=sqlite3://var/tmp/filename.sqlite3
+
+# To use the memory
+SQS_QUEUE_URL=sqlite3:memory
```

Comment:
Shouldn't this one be `sqlite3://:memory:` or `sqlite3::memory:`? 🤔

Reply:
@linkyndy we chose this "pattern" in [this discussion](https://github.com/WeTransfer/sqewer/pull/69#discussion_r541146809)

But it can be anything because there is an IF to detect it in `Sqewer::LocalConnection.parse_queue_url`

wdyt?

Reply:
I was just looking for some "standard" related to this and that's what I managed to find; I also remember using `sqlite://` before. But if it works like this and you prefer it, I'm okay with it 😊